components/checkbox/stories/template.js

```diff
@@ -90,3 +90,68 @@ export const Template = ({
 		</label>
 	`;
 };
+
+export const CheckboxGroup = (args, context) => html`
+	<div style="display: flex; flex-direction: column; padding: 16px">
+		${Template({
+			...args,
+			context,
+			iconName: undefined,
+		})}
+		${Template({
+			...args,
+			context,
+			isChecked: true,
+		})}
+		${Template({
+			...args,
+			context,
+			isIndeterminate: true,
+		})}
+		${Template({
+			...args,
+			context,
+			isDisabled: true,
+		})}
+		${Template({
+			...args,
+			context,
+			label: "Checkbox with an extraordinarily long label. Please don't do this but if you did, it should wrap text when it gets longer than the container that houses the checkbox with the unacceptably long label",
+      customStyles: { "max-inline-size": "200px" },
+		})}
+	</div>
+`;
+
+export const AllVariantsCheckboxGroup = (args, context) => {
```

Comment:
Can you rework this to go through the Variant or Container function so we're not hardcoding typography into groupings? It also standardizes our layouts and spacing.

Reply:
@castastrophe I was keeping my eye on the changes in #2912 that affect the 4 components in this branch. I was anticipating that branch getting merged first, since I understood it to be a higher priority. I haven't touched this lately because I was trying not to introduce too many rebase conflicts with your work over there and the new `test` files.

To answer your question- I definitely can rework this. BUT- do you have a preference on me waiting for #2912 to merge or do you think I should just go for it and use `Variants()` on this branch?

Reply:
Okey dokey, here are the updates!

breadcrumbs has some additional stories added to its `Variants` testing grid: https://github.com/adobe/spectrum-css/pull/2793/commits/45ba0c8ec3c32ba995ce7f318d9c62661d76a3c5
checkbox was refactored from `AllCheckboxVariantsGroup` to use `Variants`: https://github.com/adobe/spectrum-css/pull/2793/commits/a3305ea61c0391796d3c836af336abc705979d37
clearbutton uses `Variants`: https://github.com/adobe/spectrum-css/pull/2793/commits/43ec8b1a7a0db09f30f514e72d90aecc3e8fa522
coach indicator uses `Variants`: https://github.com/adobe/spectrum-css/pull/2793/commits/2373663e1048acbe2948c3876c077bb979dd7edb

Most of these have the `autodocs` tag removed now as well. (i.e. https://github.com/adobe/spectrum-css/pull/2793/commits/335b770b0fd3555a3811d6ac41b55a256c77293b)